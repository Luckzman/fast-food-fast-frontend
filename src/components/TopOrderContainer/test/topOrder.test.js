import React from 'react';
import { shallow } from 'enzyme';
import TopOrderContainer from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<TopOrderContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
