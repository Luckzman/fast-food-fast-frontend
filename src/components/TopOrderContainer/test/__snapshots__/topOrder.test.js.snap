// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div
  className="top-order"
>
  <div
    className="container"
  >
    <h1>
      top order
    </h1>
    <p>
      Customer's Top Most Choice
    </p>
    <div
      className="order-row"
    >
      <TopOrderCard
        description="Very testy and delicious with nice blend of fried plantain"
        img="https://res.cloudinary.com/dx0nauane/image/upload/v1552050529/image-4.jpg"
        key="1"
        pathId=""
        title="meat"
      />
      <TopOrderCard
        description="Very testy and delicious with nice blend of fried plantain"
        img="https://res.cloudinary.com/dx0nauane/image/upload/v1552050505/image-12.jpg"
        key="2"
        pathId=""
        title="plantain sauce"
      />
      <TopOrderCard
        description="Very testy and delicious with nice blend of fried plantain"
        img="https://res.cloudinary.com/dx0nauane/image/upload/v1544280158/x5cu9owt6sx4vxhdoz23.jpg"
        key="3"
        pathId=""
        title="meat"
      />
    </div>
  </div>
</div>
`;
